README.md

```diff
@@ -1 +1,54 @@
-# Hello-world
\ No newline at end of file
+# :green_book: Table of Contents
+- [:book: About the Project](#about-project)
+  - [:hammer_and_wrench: Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [:computer: Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [:busts_in_silhouette: Authors](#authors)
+- [:telescope: Future Features](#future-features)
+- [:handshake: Contributing](#contributing)
+- [:star:️ Show your support](#support)
+- [:pray: Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+# :book: [Hello Microverse project]
+**[Hello Microverse project]** is a microverse project where I learned about the best practices and used linters to to ensure clean code.
+## :hammer_and_wrench: Built With
+### Tech Stack
+**Client**
+React.js
+**Server**
+Express.js
+**Database**
+PostgreSQL
```

Comment:
- [ ] Very good work adding the stack here. Please make sure to add coherent information. You have not used React, Express, and any database. Please delete all of them and add HTML & CSS under client ✔️

Reply:
Done

Reply:
A message is being shown on my pull request "This branch has conflicts that must be resolved". When I try to resolve it , it say to merge. Should I merge? Could you please kindly give me a clue?

Reply:
@wayungi

Reply:
hello @tasnimchy96  you don't have to merge anything you only merge a PR which is approved only as you are still requesting reviews you don't need to merge anything yet you only have to resolve the conflicts